src/components/ProductList/index.tsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react'
+import ProductCard from '../ProductCard'
 
 const ProductList = () => {
+  const articles = new Array(20).fill(<ProductCard />)
```

Comment:
1. Since this is typescript, do you think we should specify type for every variables?
2. Although this is just a mocking. Shall we be more specific by adding `id` and `key` for each `ProductCard` ?
```suggestion
  const articles = new Array<JSX.Element | null>(20)
    .fill(null)
    .map((productCard: any, index: number) => (
      <ProductCard key={index} id={(index + 1).toString()} />
    ))
```